src/bootstrap-theme/forms/_content-editable.scss

```diff
@@ -0,0 +1,31 @@
+[contentEditable="true"] {
```

Comment:
As mentioned; this file doesn't correspond with https://github.com/twbs/bootstrap/blob/main/scss/_forms.scss

Reply:
Added issue for this to be dealt with later